tests/conftest.py

```diff
@@ -15,7 +15,7 @@
         'auth_type': 'ldap',
         'use_http_transport': True,
         'use_ssl': True,
-        'username': os.getenv('IMPALA_USER'),
+        'user': os.getenv('IMPALA_USER'),
```

Comment:
any change needed in profile.yml ?

Reply:
Nope. The test is expecting the key "user" instead of "username". Adapter supports both user and username keys.